lib/foundation_rails_helper/flash_helper.rb

```diff
@@ -18,29 +18,29 @@ module FlashHelper
       :warning   => :warning,
       :primary   => :primary
     }
-    def display_flash_messages(key_matching = {})
-      key_matching = DEFAULT_KEY_MATCHING.merge(key_matching)
+    def display_flash_messages(**options)
```

Comment:
@dsandstrom well we can always add more keyword arguments as well ;) 
Either way the body of the method needs to know about them, so using an options hash wouldn't decouple anything.